src/utils/FileManagementUtils.js

```diff
@@ -209,7 +209,14 @@ async function applyPendingOperationsOnFileParameters(
   }
 }
 
-const prepareToUpload = (event, clientFileDescriptor, setClientFileDescriptor) => {
+const isFileFormatValid = (fileMIMEType) => {
+  const acceptedFileTypes = [...VALID_MIME_TYPES];
+
+  if (acceptedFileTypes.length === 0) return true;
+  return VALID_MIME_TYPES.includes(fileMIMEType);
```

Comment:
Why do we create a copy of the Array here?

If we don't need this copy, we can rewrite the function like this:
```
const isFileFormatValid = (fileMIMEType) => VALID_MIME_TYPES.length === 0 || VALID_MIME_TYPES.includes(fileMIMEType);
```